Cubical/Codata/Conat/Base.agda

```diff
@@ -0,0 +1,22 @@
+{-# OPTIONS --cubical --safe --guardedness #-}
+module Cubical.Codata.Conat.Base where
+
+open import Cubical.Data.Unit
+open import Cubical.Data.Sum
+
+open import Cubical.Core.Everything
+
+record Conat : Set
+Prev = Unit ⊎ Conat
+record Conat where
+  coinductive
+  constructor conat
+  field prev : Prev
```

Comment:
Here I would stick to `force` for the field, because indeed it's not the previous natural number that you are projecting out.

Reply:
Looks good!